pkg/osm/ingress.go

```diff
@@ -0,0 +1,26 @@
+package osm
+
+// SupportedIngress maintains a mapping of OSM version to supported Ingress resource versions.
+var SupportedIngress = map[ControllerVersion][]IngressVersion{
+	"v0.5": {
+		// Source: https://github.com/openservicemesh/osm/blob/release-v0.5/pkg/ingress/client.go#L6
+		"extensions/v1beta1",
+	},
+	"v0.6": {
+		// Source: https://github.com/openservicemesh/osm/blob/release-v0.6/pkg/ingress/client.go#L6
+		"extensions/v1beta1",
+	},
+	"v0.7": {
+		// Source: https://github.com/openservicemesh/osm/blob/release-v0.7/pkg/ingress/client.go#L6
+		"extensions/v1beta1",
+	},
+	"v0.8": {
+		// Source: https://github.com/openservicemesh/osm/blob/release-v0.8/pkg/ingress/client.go#L6
+		"networking/v1beta1",
+	},
+	"v0.9": {
+		// Source: https://github.com/openservicemesh/osm/blob/release-v0.9/pkg/ingress/client.go#L6-L7
+		"networking/v1",
+		"networking/v1beta1",
+	},
+}
```

Comment:
In a future release, dependency on k8s ingress will be removed. Would you mind adding a TODO or creating a GH issue so we remember to come back to this later.

Reply:
@shashankram Not sure what you mean "come back to this later".
I don't anticipate we will go back retroactively and remove `networking/v1beta1` from `v0.7` - so this is here to stay.